### DISCUSSION TITLE
导入语音时RuntimeError: Numpy is not available

### DISCUSSION THREAD

**Discussion by sun-linhui** [babysor/MockingBird#720] [Q&A]
新手小白不知怎么解决，路过的各位帮帮忙
在导入语音时遇到的错误

E:\Graduate Programs\MockingBird\MockingBird-main>python demo_toolbox.py
Arguments:
    datasets_root:          None
    vc_mode:                False
    enc_models_dir:         encoder\saved_models
    syn_models_dir:         synthesizer\saved_models
    voc_models_dir:         vocoder\saved_models
    extractor_models_dir:   ppg_extractor\saved_models
    convertor_models_dir:   ppg2mel\saved_models
    cpu:                    False
    seed:                   None
    no_mp3_support:         False

Warning: you did not pass a root directory for datasets as argument.
The recognized datasets are:
        LibriSpeech/dev-clean
        LibriSpeech/dev-other
        LibriSpeech/test-clean
        LibriSpeech/test-other
        LibriSpeech/train-clean-100
        LibriSpeech/train-clean-360
        LibriSpeech/train-other-500
        LibriTTS/dev-clean
        LibriTTS/dev-other
        LibriTTS/test-clean
        LibriTTS/test-other
        LibriTTS/train-clean-100
        LibriTTS/train-clean-360
        LibriTTS/train-other-500
        LJSpeech-1.1
        VoxCeleb1/wav
        VoxCeleb1/test_wav
        VoxCeleb2/dev/aac
        VoxCeleb2/test/aac
        VCTK-Corpus/wav48
        aidatatang_200zh/corpus/dev
        aidatatang_200zh/corpus/test
        aishell3/test/wav
        magicdata/train
Feel free to add your own. You can still use the toolbox by recording samples yourself.
Loaded encoder "pretrained.pt" trained to step 1594501
Traceback (most recent call last):
  File "E:\Graduate Programs\MockingBird\MockingBird-main\toolbox\__init__.py", line 112, in <lambda>
    func = lambda: self.load_from_browser(self.ui.browse_file())
  File "E:\Graduate Programs\MockingBird\MockingBird-main\toolbox\__init__.py", line 192, in load_from_browser
    self.add_real_utterance(wav, name, speaker_name)
  File "E:\Graduate Programs\MockingBird\MockingBird-main\toolbox\__init__.py", line 216, in add_real_utterance
    embed, partial_embeds, _ = encoder.embed_utterance(encoder_wav, return_partials=True)
  File "E:\Graduate Programs\MockingBird\MockingBird-main\encoder\inference.py", line 164, in embed_utterance
    partial_embeds = embed_frames_batch(frames_batch)
  File "E:\Graduate Programs\MockingBird\MockingBird-main\encoder\inference.py", line 62, in embed_frames_batch
    frames = torch.from_numpy(frames_batch).to(_device)
RuntimeError: Numpy is not available

**Comment by wei-z-git** [comment]
依赖没安装
`pip3 install -r ./requirements.txt`
